### PR TITLE
Spacing token descriptions (global and alias)

### PR DESCRIPTION
**Pull request by raoufswe**
- [Added] spacing token descriptions (global and alias)
- [Added] `component-updates_narrow.svg` to the designers `best practices` page
- [Added] full stops for list items in the `our vision` pag
- [Add] missing heading for the `faq` page
- [Fixed] unclear SVGs used in the designers `how we support you` page
- [Removed] `Who works on PIE?` section in the faq page

### CODE REVIEW

File: apps/pie-microsite/src/assets/styles/components/_tokensTable.scss [hunk 1]
```diff
@@ -248,6 +248,18 @@ $tokensTable-item-vertical-padding: f.spacing(e);
...
+
+    @include f.media('>=wide') {
+        margin-top: 0;
+
+        & .c-tokensTable-token {
+            margin-left: f.spacing(b);
+        }
+    }
+}
+
```
Comment (jamieomaguire) [inline apps/pie-microsite/src/assets/styles/components/_tokensTable.scss:262]: @raoufswe could we chat about this? Not quite sure why this has been added as the previous styles for the global token used were signed off by jordi. Looking on the PR preview, the margin seems to be incorrect on this PR

Screenshot 2023-02-07 at 08 47 55
Screenshot 2023-02-07 at 08 47 46

**Review by jamieomaguire — CHANGES_REQUESTED** [review]
Just setting to request changes so we can catch up about the global tokens stuff :)

**Review by FayeCarter — DISMISSED** [review]

**Review by fernandofranca — APPROVED** [review]

**Review by jamieomaguire — CHANGES_REQUESTED** [review]
we've updated the spacing in figma - can explain in a message or on a call but good to align with the other tokens

**Review by LTurns — APPROVED** [review]

**Review by fernandofranca — APPROVED** [review]

**Review by jamieomaguire — APPROVED** [review]

**Comment by jamieomaguire** [comment]
LGTM - awesome work @raoufswe 🎉

**Pull request merged**